README.md

```diff
@@ -99,7 +99,7 @@ In order the TFT firmware is able to provide all of its functionalities/features
 `M114_DETAIL` (in Configuration_adv.h)<br>
 `REPORT_FAN_CHANGE` (in Configuration_adv.h)<br>
 
-**Options to support printing from onboard SD:**
+**Options to support printing from onboard media:**
```

Comment:
you should apply that change also on top of all the 3 config.ini files

Reply:
I am not sure what do you suggest here. Can you be please more specific?

Reply:
the same text is provided at the beginning of each config.ini file. So also that text should be modified:

`# Options to support printing from onboard SD:
`
to

`# Options to support printing from onboard media:
`

Reply:
I cannot find "# Options to support printing from onboard SD:" left unchanged.

Reply:
oh sorry, my mistake. I didn't see that change

Reply:
No problem, I thought my VSCode is messing with me :)